redditdownloader/processing/handlers/redgifs.py

```diff
@@ -0,0 +1,37 @@
+import re
+from processing.wrappers import http_downloader
+
+tag = 'redgifs'
+order = 1
+
+""" 
+	This Handler attempts to convert Gyfcat links into direct video links.
+"""
+
+format_opts = ["webmUrl", "mp4Url", "gifUrl"]
+
+
+def handle(task, progress):
+	url = task.url
+	try:
+		redgif_id = re.match(r'.*/(.*?)/?$', url).group(1)
```

Comment:
Is there any way we can make this pattern more restrictive so it doesn't attempt to download from irrelevant urls?